Makefile.js

```diff
@@ -159,7 +159,8 @@ function execSilent(cmd) {
  * @private
  */
 function generateBlogPost(releaseInfo) {
-    const output = ejs.render(cat("./templates/blogpost.md.ejs"), releaseInfo),
+    const ruleList = ls("lib/rules").map(ruleFileName => ruleFileName.replace(/\.js$/, "")).sort((ruleA, ruleB) => ruleB.length - ruleA.length);
```

Comment:
Oh, and I thought of one more thing-- any point in maybe using `path.basename` instead of removing `.js`? In case we ever decided to use a different extension for some crazy reason, I mean.